README.md

```diff
@@ -1 +1,190 @@
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests ](#run-tests-)
+    - [Deployment ](#deployment-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-rails] <a name="about-project"></a>
+
+**[Hello-rails]()** This is an App that demonstrates using rails for backend and react for frontend in two separate applications.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>RUBY</li>
+  </ul>
+  <ul>
+    <li>RUBY ON RAILS</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Provides an API to display random greetings]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li>The current version of ruby</li>
+    <li>To have Git installed on your local machine</li>
+    <li>To have postgresql installed on your local machine</li>
+    <li>To be connected to a databse</li>
+    <li>gem </li>
+    <li>An editor such as Visual Studio Code</li>
+  </ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repositories to your desired folders for backend and frontend respectively:
+
+<ul>
+<li>run git clone "https://github.com/prius2055/hello-world-backend.git"</li>
+<li>run git clone "https://github.com/prius2055/hello-world-frontend.git"</li>
+</ul>
+  
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+To install this project, run
+
+```sh
+
+Bundle install
+
+```
+
+```sh
+
+gem install rails
+
+```
+
+### Run tests <a name="run-tests"></a>
+
+To run tests, run the following command:
+
+Run
+
+```sh
+
+ "rubocop --fix"
+
+```
+
+to fix linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using:
+
+- github pages
+
+```sh
+- npm run deploy
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Prince**
+
+- GitHub: [@prius2055](https://github.com/prius2055)
+- Twitter: [@prius2055](https://www.twitter.com/prius2055)
+- LinkedIn: [@prius2055](https://www.linkedin.com/in/princenwuke)
+
+  👤 **Mahabub Alam**
```

Comment:
- [ ] Kindly remove this second author since this is a solo project.

Reply:
Thanks a lot